### DISCUSSION TITLE
Using secrets from a reusable workflow repository

### DISCUSSION THREAD

**Discussion by philipsabri** [community/community#50728] [Actions] [answered]
We have a situation in which we want to store a token as a secret within a reusable workflow repository and utilize it in a workflow that is being called by another repository. This ensures that the token can only be used in a specific manner configured within the reusable workflow repository. I dont want to use secrets from the calling repository either.

Organization secrets are not suitable because they could be accessed by "anyone," potentially leading to unwanted usage.

Have anyone solved this somehow or have any workaround?

**Comment by juliandunn** [comment, marked as answer]
Is this what you are after? We are working on it: https://github.com/github/roadmap/issues/636

**Reply by matherm-aboehm** [reply]
@juliandunn this was suddenly removed from the roadmap.
I can still see demand for that: https://github.com/orgs/community/discussions/147339
Are there any plans to add that back to the roadmap?
